src/TrapException.cs

```diff
@@ -96,11 +135,33 @@ private TrapException(string message, IReadOnlyList<TrapFrame> frames) : base(me
             Frames = frames;
         }
 
+        /// <summary>
+        /// Returns an optional `TrapCode` that corresponds to why this trap happened. 
+        /// Note that `Unknown` may be returned for manually created traps which do 
+        /// not have an associated code with them.
+        /// </summary>
+        /// <param name="trap"></param>
+        /// <returns></returns>
+
```

Comment:
I think this can be removed as this project doesn't usually use doc comments on private methods and the method, I think, is pretty small and self-explanatory.

```suggestion
```